lib/rules/no-unused-vars.js

```diff
@@ -18,26 +18,49 @@ module.exports = {
       category: 'essential',
       url: 'https://eslint.vuejs.org/rules/no-unused-vars.html'
     },
-    fixable: null,
-    schema: []
+    fixable: 'code',
+    schema: [
+      {
+        'type': 'object',
+        'properties': {
+          'varIgnorePattern': {
+            'type': 'string'
+          }
+        },
+        'additionalProperties': false
+      }
+    ]
   },
 
   create (context) {
+    const option = context.options[0] || { varIgnorePattern: '^_' }
+    const pattern = option['varIgnorePattern'] || '^_'
+    // only use for construct a regularExpression
+    // eslint-disable-next-line no-eval
+    const regExp = eval(`/${pattern}/`)
```

Comment:
Do not use `eval()`.

You can handle it in the same way as ESLint's no-unused-vars.

https://github.com/eslint/eslint/blob/master/lib/rules/no-unused-vars.js#L106